src/api/helpers/date/index.ts

```diff
@@ -42,3 +42,16 @@ export const getYesterdayDay = (): number => {
 
   return result;
 };
+
+/**
+ * getThirtyDaysBefore
+ * Get 30 days before now and return the timestamp
+ * @returns {Date} 30 days before today
+ */
+export const getThirtyDaysBefore = (): Date => {
```

Comment:
Could we rename this to be `getThirtyDaysBeforeNow`, otherwise feels like "before" could be a param